config/locales/en/transition_landing_page.yml

```diff
@@ -9,7 +9,9 @@ en:
     take_action_title: Take action
     take_action_title_variant_B: Make sure you're ready
     take_action_text: |
-      <p>Answer a few questions to get a personalised list of actions for you, your family, and your business. Then sign up for emails to get updates when things change.</p>
+      <p class="take_action_test_class">Answer a few questions to get a personalised list of actions for you, your family, and your business. Then sign up for emails to get updates when things change.</p>
```

Comment:
Could we move `<p class="take_action_test_class">` tags into the view? It seems the same for both (though would need to check the Welsh version too).
That would mean we could drop the `html_safe` call and sleep well at night. I think we'd be able to regenerate the brakeman config too, as it wouldn't need an exclusion any more.

Reply:
Ah yes that would be nicer, will give that a go.